### PR TITLE
Make Armor shop message say equipped

### PR DESCRIPTION
**Pull request by JT122406**
and not maxed as that does not make sense for armor
Also available to be changed

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java [hunk 1]
```diff
@@ -34,10 +34,12 @@
 import com.andrei1058.bedwars.shop.ShopCache;
 import com.andrei1058.bedwars.shop.quickbuy.PlayerQuickBuyCache;
 import com.andrei1058.bedwars.shop.quickbuy.QuickBuyElement;
+import net.minecraft.server.v1_8_R3.ItemArmor;
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java]: you are breaking multi version support here man

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java [hunk 2]
```diff
@@ -34,10 +34,12 @@
...
 import org.bukkit.Bukkit;
 import org.bukkit.ChatColor;
 import org.bukkit.Material;
 import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.craftbukkit.v1_8_R3.inventory.CraftItemStack;
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java]: same here

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java [hunk 3]
```diff
@@ -260,7 +262,11 @@ public ItemStack getItemStack(Player player, ShopCache shopCache) {
             String buyStatus;
 
             if (isPermanent() && shopCache.hasCachedItem(this) && shopCache.getCachedItem(this).getTier() == getContentTiers().size()) {
-                buyStatus = getMsg(player, Messages.SHOP_LORE_STATUS_MAXED);
+                if (!(CraftItemStack.asNMSCopy(i).getItem() instanceof ItemArmor)){
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java]: this must be done through **BedWars.nms** https://github.com/andrei1058/BedWars1058/blob/master/bedwars-api/src/main/java/com/andrei1058/bedwars/api/server/VersionSupport.java#L106

**Review by andrei1058 — CHANGES_REQUESTED** [review]

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java (hunk 3 above)
Comment (JT122406) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/shop/main/CategoryContent.java]: done

**Pull request merged**